docs/how-to-use-relational-entity-storage.md

```diff
@@ -0,0 +1,74 @@
+---
+title: How to use relational entity storage
+slug: /how-to-use-relational-entity-storage
+license: "Copyright 2024 Datastrato Pvt Ltd.
+This software is licensed under the Apache License version 2."
+---
+
+## Introduction
+
+Gravitino supports using `Relational Entity Storage` to store metadata after version `0.5.0`.  
```

Comment:
I suggest you add more description about what is `Relational Entity Storage` and what kind of `Relational Entity Storage` Gravitino supports.